docs/api/apiv3/components/schemas/file_link_model.yml

```diff
@@ -30,6 +27,17 @@ properties:
     readOnly: true
   originData:
     $ref: '#/components/schemas/File_Link_Origin_DataModel'
+  _embedded:
+    type: object
+    required:
+      - storage
+      - container
+    properties:
+      storage:
+        $ref: '#/components/schemas/StorageModel'
+      container:
+        oneOf:
+          - $ref: '#/components/schemas/Work_PackageModel'
```

Comment:
Both of those are `readOnly: true`

Reply:
put `readOnly` on the `_embedded` itself, as it is not feasible to put it next to a `$ref`.